TACC/Core-Portal
# Add Elasticsearch caching for TAS allocations.

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [FP-565](https://jira.tacc.utexas.edu/browse/FP-565)

## Summary of Changes: ##

## Testing Steps: ##
1. execute `portal.libs.elasticsearch.indexes.setup_allocations_index()`
2. Navigate to Allocations in the portal and curl `http://cep.dev:9200/frontera-dev-allocations/_search` to ensure that allocations have been indexed.

## UI Photos:

## Notes: ##
